BrainPortal/app/models/signup.rb

```diff
@@ -60,15 +60,11 @@ def generate_token
     self.confirm_token = tok
   end
 
-  def full #:nodoc:
+  def full_name #:nodoc:
     "#{title} #{first} #{middle} #{last}".strip.gsub(/  +/, " ")
   end
 
-  def user_full #:nodoc:
-    "#{first} #{last}".strip.gsub(/  +/, " ")
-  end
-
-  alias full_name full
+  # alias full_name full
```

Comment:
Why leave the alias commented out? Just delete it. Not only that, but if the alas was re-insterted now, it would crash, because there is no longer a method called `full`. So the alias itself has been made invalid by your code changes.

Reply:
Also `BUG!` You forgot to change the method call in `BrainPortal/app/views/cbrain_mailer/signup_notify_admin.text.erb`